migrations/src/9_deploy_razor_and_faucet.js

```diff
@@ -0,0 +1,33 @@
+const { NETWORK, SCHELLING_COIN_ADDRESS } = process.env;
+const {
+  deployContract,
+  appendDeploymentFile,
+  readOldDeploymentFile,
+} = require('../migrationHelpers');
+
+const { BigNumber } = ethers;
+const initialSupply = (BigNumber.from(10).pow(BigNumber.from(27)));
+const deployRAZORAndFaucet = async () => {
+  if (NETWORK !== 'mainnet' && SCHELLING_COIN_ADDRESS === '') {
```

Comment:
Please update process.env as well
Why to use  SCHELLING_COIN_ADDRESS as variable name
```
 (NETWORK !== 'mainnet' && SCHELLING_COIN_ADDRESS === '')
 ```

Reply:
yeah